guides/manual-installation.md

```diff
@@ -0,0 +1,279 @@
+
+```bash
+firewall-cmd --zone=public --add-port=15672/tcp --permanent
+firewall-cmd --reload
+```
+
+### Install container runtime
+
+First we need to enable the containers module:
+
+```bash
+SUSEConnect --product sle-module-containers/15.5/x86_64
+```
+
+Then we can install docker:
+
+```bash
+zypper install docker
+```
+
+Enable and start Docker:
+
+```bash
+systemctl enable --now docker
+```
+
+### Deploy trento-wanda and trento-web components on docker:
+
+#### Create the secrets
+
+```bash
+WANDA_SECRET_KEY_BASE=$(openssl rand -out /dev/stdout 48 | base64)
+TRENTO_SECRET_KEY_BASE=$(openssl rand -out /dev/stdout 48 | base64)
+ACCESS_TOKEN_ENC_SECRET=$(openssl rand -out /dev/stdout 48 | base64)
+REFRESH_TOKEN_ENC_SECRET=$(openssl rand -out /dev/stdout 48 | base64)
+
+```
+
+#### Create a dedicated docker network for trento
+
+```bash
+docker network create trento-net
+```
+
+#### Install trento-wanda on docker:
+
+```bash
+docker run -d --name wanda \
+    -p 4001:4000 \
+    --network trento-net \
+    --add-host "host.docker.internal:host-gateway" \
+    -e CORS_ORIGIN=localhost \
+    -e SECRET_KEY_BASE=$WANDA_SECRET_KEY_BASE \
+    -e ACCESS_TOKEN_ENC_SECRET=$ACCESS_TOKEN_ENC_SECRET \
+    -e AMQP_URL=amqp://guest:guest@host.docker.internal \
+    -e DATABASE_URL=ecto://postgres:postgres@host.docker.internal/postgres \
+    --entrypoint /bin/sh \
+    registry.suse.com/trento/trento-wanda:1.2.0 \
+    -c "/app/bin/wanda eval 'Wanda.Release.init()' && /app/bin/wanda start"
+```
+
+#### Install trento-web on docker:
+
+> Note: Be sure to change the `ADMIN_USERNAME` and `ADMIN_PASSWORD` environment variables to your desired values.
+
+> Note: Be sure to change the `GRAFANA_PASSWORD` (and potentially `GRAFANA_USER` if you changed it) environment variable to the password you set for the grafana admin user in the grafana installation step.
+
+```
+docker run -d \
+    -p 4000:4000 \
+    --name trento-web \
+    --network trento-net \
+    --add-host "host.docker.internal:host-gateway" \
+    -e AMQP_URL=amqp://guest:guest@host.docker.internal \
+    -e DATABASE_URL=ecto://postgres:postgres@host.docker.internal/trento \
+    -e EVENTSTORE_URL=ecto://postgres:postgres@host.docker.internal/trento_event_store \
+    -e ENABLE_ALERTING=false \
+    -e GRAFANA_PUBLIC_URL='http://host.docker.internal:3000' \
+    -e GRAFANA_API_URL='http://host.docker.internal:3000/api' \
+    -e PROMETHEUS_URL='http://host.docker.internal' \
+    -e SECRET_KEY_BASE=$TRENTO_SECRET_KEY_BASE \
+    -e ACCESS_TOKEN_ENC_SECRET=$ACCESS_TOKEN_ENC_SECRET \
+    -e REFRESH_TOKEN_ENC_SECRET=$REFRESH_TOKEN_ENC_SECRET \
+    -e ADMIN_USERNAME='admin' \
+    -e ADMIN_PASSWORD='test1234' \
+    -e GRAFANA_PASSWORD='trento' \
+    -e ENABLE_API_KEY='true' \
+    --entrypoint /bin/sh \
+    registry.suse.com/trento/trento-web:2.2.0 \
+    -c "/app/bin/trento eval 'Trento.Release.init()' && /app/bin/trento start"
+```
```

Comment:
just an idea, add some commands to ensure that the containers are running fine. 
Also how to actually access trento web would be great!